src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -61,38 +61,60 @@ public void MefCtor_CheckIsExported()
         }
 
         [TestMethod]
-        public void SynchronizeWithServer_FailureToCheckServerConnection_CriticalException_ExceptionThrown()
```

Comment:
Redundant test - deleted. There were multiple tests that checked critical/non-critical errors. I've kept one of each.